src/content/en/updates/2018/01/paintapi.md

```diff
@@ -0,0 +1,278 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Houdini’s CSS Paint API allows you to programmatically draw CSS image resources.
+
+{# wf_updated_on: 2018-01-18 #}
+{# wf_published_on: 2018-01-18 #}
+{# wf_tags: css,style,houdini,javascript #}
+{# wf_featured_image: /web/updates/images/2018/01/paintapi/houdinidiamond.png #}
+{# wf_featured_snippet: Houdini’s CSS Paint API allows you to programmatically draw CSS image resources. #}
+{# wf_blink_components: Blink>CSS #}
+
+
+# CSS Paint API {: .page-title }
+
+{% include "web/_shared/contributors/surma.html" %}
+
+## New possibilities in Chrome 65
```

Comment:
Is this really shipping in 65? If so we need to update https://www.chromestatus.com/features/5685444318593024 and get this in the beta blog post draft asap.

Reply:
I was just about to do that just wanted to get confirmation from @bfgeek (which I just got). Gonna update the feature.

Reply:
If it is in 65, please also add `chrome65` to `wf_tags`